src/dgipy/dgidb.py

```diff
@@ -122,69 +150,123 @@ def get_interactions(
     :param use_pandas: boolean for whether pandas should be used to format response
     :param immunotherapy: filter option for results that are used in immunotherapy
     :param antineoplastic: filter option for results that are part of antineoplastic regimens
-    :param sourcedbname: filter option for specific databases of interest
-    :param pmid: filter option for specific PMIDs:
-    :param interactiontype: filter option for specific interaction types
+    :param source: filter option for specific database of interest
+    :param pmid: filter option for specific PMID
+    :param interaction_type: filter option for specific interaction types
     :param approved: filter option for approved interactions
     :param api_url: API endpoint for GraphQL request
     :return: interaction results for terms in either a dataframe or a json object
     """
```

Comment:
we should make sure that theres no case sensitivity issues on the resolver side as well for parameters through graphql. iirc one resolver had an issue with that while you were out and we had to change it

Reply:
oh I believe it